### PR TITLE
ordering client dropdown

### PR DESCRIPTION
**Pull request by nishant22029**
Targets #2773 

### Description
In this PR  ,I updated the query that is used to fetch client details in a way that is to order the client's result by client name in ascending order.
 The client list is not ordered by name which is causing trouble, if we sort the list by client name in ascending order then it would be easier to find the client for that list.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Http/Controllers/ProjectController.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Modules\Project\Http\Controllers;
 
+use Google\Service\AnalyticsData\OrderBy;
```
Comment (GauravGusain98) [inline Modules/Project/Http/Controllers/ProjectController.php:5]: This is not needed. Why it was imported?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Looks good minor suggestion

File: Modules/Project/Http/Controllers/ProjectController.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace Modules\Project\Http\Controllers;
 
+// use Google\Service\AnalyticsData\OrderBy;
```
Comment (GauravGusain98) [inline Modules/Project/Http/Controllers/ProjectController.php:5]: Commenting code is not a good practice. Remove this line as it was not there earlier.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Check the comment

File: Modules/Project/Http/Controllers/ProjectController.php (hunk 2 above)
Comment (nishant22029) [reply Modules/Project/Http/Controllers/ProjectController.php:5]: ok sir

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**